scripts/qa_crowbarsetup.sh

```diff
@@ -4439,6 +4439,12 @@ function oncontroller_testpostupgrade
 {
     # retrieve the ping results
     local fips=$(openstack floating ip list -f value -c "Floating IP Address")
+
+    # remove manila-service fip from list
+    manila_vm=$(openstack server list --all-projects -f value | grep manila-service | awk '{ print $1 }' )
```

Comment:
Isnt the manila instance always called `manila-service` ? Seems like this line could be removed and instead set `manila-service` in the line below instead of `$manila_vm`, especially if we are still gonna hardcode the `grep manila-service` in here.